examples/classifying_shapes.ipynb

```diff
@@ -21,11 +21,11 @@
    "outputs": [],
    "source": [
     "# Importing libraries\n",
-    "import giotto as go\n",
-    "import giotto.time_series as ts\n",
-    "import giotto.diagrams as diag\n",
-    "import giotto.homology as hl\n",
-    "from giotto.diagrams import PersistenceEntropy, BettiCurve, PersistenceLandscape, HeatKernel\n",
+    "import glearn as go\n",
```

Comment:
An interesting question is whether `gl` would make more sense for the alias, but not worth the effort now.